### ISSUE TITLE
Remove calendar events table from database

### ISSUE THREAD

**Issue by rlgreen91** [EBWiki/EBWiki#4047]
Remove the current `calendar_events` table from the database.  

Dev Note: You do not have to remove the associated model file.

**Comment by luhova** [comment]
Hello, I would like to take my hands on this issue as part of my Hacktoberfest contribution this year. 

P.S. Funny thing - two years ago I contributed to your project by creating the `calendar_events` table and now here we are :sweat_smile:

**Comment by rlgreen91** [comment]
Hi @luhova, welcome back and thanks for picking this up!  That's funny about adding the table during a previous Hacktoberfest 🤣 We were going to use a gem and another library to do the calendar events originally, but we decided to go in a different direction.  I thought it would be easiest to just drop the table we have now completely and start from scratch with a new one.

If you like, once the PR for this ticket gets merged, I can assign the follow-up issue to create the new table to you.  That'll be for Hacktoberfest as well, I just haven't labeled it yet since this one has to be done first.

**Comment by luhova** [comment]
Hi @rlgreen91, yes, I saw the PR you are talking about, I would gladly take my hands on it too. You can assign it to me. :raised_hands:
